dot_config/nvim/lua/tap/lsp/servers/tsserver.lua

```diff
@@ -17,13 +19,17 @@ end
 
 local module = {}
 
-local server_name = "typescript"
-local lspconfig_name = "tsserver"
+function module.patch_install()
+    lsp_utils.patch_lsp_installer("tsserver", installers.pipe {
+        -- Copy of default tsserver installer but pin ts-ls to 0.6.5
+        npm.packages {"typescript-language-server@0.6.5", "typescript"}
```

Comment:
I can remove this pinning now https://github.com/typescript-language-server/typescript-language-server/pull/314 is merged